packages/pods-core/src/builtin/MarkdownPod.ts

```diff
@@ -336,6 +333,21 @@ export class MarkdownExportPod extends ExportPod {
         return fs.writeFile(fpath, body);
       })
     );
+
+    // Export Assets
+    vaults.forEach(async (vault) => {
```

Comment:
this is an async function and will return an array of promises. this means this can finish after the function returns. we need to add an `await Promise.all(vaults.forEach(...)`

Reply:
good catch